pkg/middlewares/compress/compress.go

```diff
@@ -76,7 +76,13 @@ func New(ctx context.Context, next http.Handler, conf dynamic.Compress, name str
 	}
 
 	var err error
-	c.brotliHandler, err = c.newBrotliHandler()
+
+	c.zstdHandler, err = c.newCompressionHandler(Zstandard, name)
+	if err != nil {
+		return nil, err
+	}
+
+	c.brotliHandler, err = c.newCompressionHandler(Brotli, name)
```

Comment:
Hi, I'm wondering why we should prioritise zstd over other encodings?
I understand there's no "good choice" since we don't parse quality values, but it seems for the same reason we could give zstd the least priority and avoid breaking/altering compress middleware behaviour. WDYT?